services/api/src/routes/auth.js

```diff
@@ -33,6 +33,12 @@ router
         ...ctx.request.body,
       });
 
+      await AuditLog.append('registered', ctx, {
```

Comment:
I'm not sure we want bedrock out of the box to actually use AuditLog as I feel like most projects won't need it... the same goes for the routes... the facility should just be available. Actually it feels more like a plugin but I think we decided against that for some reason.

Reply:
I am fine with removing it, this started out mainly to make it easier to see how to use it. But then I realized this would something i would have enabled by default. E.g. anything around users could be good track for security. 
 
But yeah lets see what @dominiek says

Reply:
It takes up a lot of space in the code and is meaningless for most people approaching bedrock out of the box... I would much rather our code show off how simple it is.